2022-08-11 23:36:27 INFO run.py:219 GNS3 server version 2.2.33.1
2022-08-11 23:36:27 INFO run.py:221 Copyright (c) 2007-2022 GNS3 Technologies Inc.
2022-08-11 23:36:27 INFO run.py:224 Config file /home/h4ck4u/.config/GNS3/2.2/gns3_server.conf loaded
2022-08-11 23:36:27 WARNING run.py:230 Local mode is enabled. Beware, clients will have full control on your filesystem
2022-08-11 23:36:27 INFO run.py:237 HTTP authentication is enabled with username 'admin'
2022-08-11 23:36:27 INFO run.py:243 Running with Python 3.10.5 and has PID 45674
2022-08-11 23:36:27 INFO run.py:79 Current locale is en_US.UTF-8
2022-08-11 23:36:28 INFO web_server.py:318 Starting server on localhost:3080
2022-08-11 23:36:28 INFO __init__.py:63 Load controller configuration file /home/h4ck4u/.config/GNS3/2.2/gns3_controller.conf
2022-08-11 23:36:28 INFO __init__.py:67 Controller is starting
2022-08-11 23:36:28 INFO compute.py:64 Create compute local
2022-08-11 23:36:28 INFO compute.py:364 Connecting to compute 'local'
2022-08-11 23:36:28 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:28 +0000] "GET /v2/compute/capabilities HTTP/1.1" 200 557 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:28 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:28 +0000] "GET /v2/version HTTP/1.1" 200 232 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:36:28 INFO notification_handler.py:50 New client has connected to compute WebSocket
2022-08-11 23:36:28 INFO compute.py:434 Connected to compute 'local' WebSocket 'http://localhost:3080/v2/compute/notifications/ws'
2022-08-11 23:36:28 INFO notification_handler.py:73 New client has connected to controller WebSocket
2022-08-11 23:36:28 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:28 +0000] "GET /v2/version HTTP/1.1" 200 232 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:36:28 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:28 +0000] "GET /v2/templates HTTP/1.1" 200 3605 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:36:28 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:28 +0000] "GET /v2/computes HTTP/1.1" 200 1029 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:36:28 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:28 +0000] "GET /v2/appliances?symbol_theme=Classic HTTP/1.1" 200 811804 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:36:28 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:28 +0000] "GET /v2/projects HTTP/1.1" 200 5269 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:36:29 INFO project.py:78 Project b963e8c3-589d-4a9c-b535-bbf1907b6ef0 with path '/home/h4ck4u/Downloads/laRota' created
2022-08-11 23:36:29 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects HTTP/1.1" 201 304 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO __init__.py:303 Creating new hypervisor localhost:47777 with working directory /home/h4ck4u/Downloads/laRota/project-files/dynamips
2022-08-11 23:36:29 INFO hypervisor.py:127 Starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_i1_log.txt', '-H', 'localhost:47777']
2022-08-11 23:36:29 INFO hypervisor.py:129 Dynamips process logging to /home/h4ck4u/Downloads/laRota/project-files/dynamips/dynamips_i1_stdout.txt
2022-08-11 23:36:29 INFO hypervisor.py:136 Dynamips process started PID=45701
2022-08-11 23:36:29 INFO __init__.py:305 Hypervisor localhost:47777 has successfully started
2022-08-11 23:36:29 INFO dynamips_hypervisor.py:92 Connected to Dynamips hypervisor on localhost:47777 after 0.2951 seconds
2022-08-11 23:36:29 INFO router.py:222 Router c3725 "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba] has been created
2022-08-11 23:36:29 INFO router.py:952 Router "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba]: auto delete disks enabled
2022-08-11 23:36:29 INFO router.py:903 Router "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba]: disk0 updated from 16MB to 0MB
2022-08-11 23:36:29 INFO router.py:489 Router "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba]: has a new IOS image set: "/home/h4ck4u/GNS3/images/IOS/c3725-adventerprisek9_ivs-mz.124-15.T14.image"
2022-08-11 23:36:29 INFO router.py:1027 Router "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba]: MAC address updated from c201.b285.0000 to c201.0eed.0000
2022-08-11 23:36:29 INFO router.py:543 Router "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba]: NVRAM updated from 112KB to 256KB
2022-08-11 23:36:29 INFO router.py:1102 Router "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba]: adapter NM-1FE-TX inserted into slot 1
2022-08-11 23:36:29 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes HTTP/1.1" 201 1200 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO __init__.py:303 Creating new hypervisor localhost:60025 with working directory /home/h4ck4u/Downloads/laRota/project-files/dynamips
2022-08-11 23:36:29 INFO hypervisor.py:127 Starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_i2_log.txt', '-H', 'localhost:60025']
2022-08-11 23:36:29 INFO hypervisor.py:129 Dynamips process logging to /home/h4ck4u/Downloads/laRota/project-files/dynamips/dynamips_i2_stdout.txt
2022-08-11 23:36:29 INFO hypervisor.py:136 Dynamips process started PID=45717
2022-08-11 23:36:29 INFO __init__.py:305 Hypervisor localhost:60025 has successfully started
2022-08-11 23:36:29 INFO dynamips_hypervisor.py:92 Connected to Dynamips hypervisor on localhost:60025 after 0.0128 seconds
2022-08-11 23:36:29 INFO router.py:222 Router c3725 "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5] has been created
2022-08-11 23:36:29 INFO router.py:952 Router "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5]: auto delete disks enabled
2022-08-11 23:36:29 INFO router.py:903 Router "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5]: disk0 updated from 16MB to 0MB
2022-08-11 23:36:29 INFO router.py:489 Router "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5]: has a new IOS image set: "/home/h4ck4u/GNS3/images/IOS/c3725-adventerprisek9_ivs-mz.124-15.T14.image"
2022-08-11 23:36:29 INFO router.py:1027 Router "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5]: MAC address updated from c202.b295.0000 to c202.0efe.0000
2022-08-11 23:36:29 INFO router.py:543 Router "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5]: NVRAM updated from 112KB to 256KB
2022-08-11 23:36:29 INFO router.py:1102 Router "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5]: adapter NM-1FE-TX inserted into slot 1
2022-08-11 23:36:29 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes HTTP/1.1" 201 1200 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO __init__.py:303 Creating new hypervisor localhost:54595 with working directory /home/h4ck4u/Downloads/laRota/project-files/dynamips
2022-08-11 23:36:29 INFO hypervisor.py:127 Starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_i3_log.txt', '-H', 'localhost:54595']
2022-08-11 23:36:29 INFO hypervisor.py:129 Dynamips process logging to /home/h4ck4u/Downloads/laRota/project-files/dynamips/dynamips_i3_stdout.txt
2022-08-11 23:36:29 INFO hypervisor.py:136 Dynamips process started PID=45733
2022-08-11 23:36:29 INFO __init__.py:305 Hypervisor localhost:54595 has successfully started
2022-08-11 23:36:29 INFO dynamips_hypervisor.py:92 Connected to Dynamips hypervisor on localhost:54595 after 0.0121 seconds
2022-08-11 23:36:29 INFO router.py:222 Router c3725 "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69] has been created
2022-08-11 23:36:29 INFO router.py:952 Router "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69]: auto delete disks enabled
2022-08-11 23:36:29 INFO router.py:903 Router "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69]: disk0 updated from 16MB to 0MB
2022-08-11 23:36:29 INFO router.py:489 Router "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69]: has a new IOS image set: "/home/h4ck4u/GNS3/images/IOS/c3725-adventerprisek9_ivs-mz.124-15.T14.image"
2022-08-11 23:36:29 INFO router.py:1027 Router "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69]: MAC address updated from c203.b2a5.0000 to c203.0f0e.0000
2022-08-11 23:36:29 INFO router.py:543 Router "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69]: NVRAM updated from 112KB to 256KB
2022-08-11 23:36:29 INFO router.py:1102 Router "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69]: adapter NM-1FE-TX inserted into slot 1
2022-08-11 23:36:29 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes HTTP/1.1" 201 1200 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO __init__.py:303 Creating new hypervisor localhost:32999 with working directory /home/h4ck4u/Downloads/laRota/project-files/dynamips
2022-08-11 23:36:29 INFO hypervisor.py:127 Starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_i4_log.txt', '-H', 'localhost:32999']
2022-08-11 23:36:29 INFO hypervisor.py:129 Dynamips process logging to /home/h4ck4u/Downloads/laRota/project-files/dynamips/dynamips_i4_stdout.txt
2022-08-11 23:36:29 INFO hypervisor.py:136 Dynamips process started PID=45749
2022-08-11 23:36:29 INFO __init__.py:305 Hypervisor localhost:32999 has successfully started
2022-08-11 23:36:29 INFO dynamips_hypervisor.py:92 Connected to Dynamips hypervisor on localhost:32999 after 0.0123 seconds
2022-08-11 23:36:29 INFO ethernet_switch.py:187 Ethernet switch "Switch1" [8eb11374-ef90-4b72-aefe-567b232d1d27] has been created
2022-08-11 23:36:29 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes HTTP/1.1" 201 1564 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO __init__.py:303 Creating new hypervisor localhost:40385 with working directory /home/h4ck4u/Downloads/laRota/project-files/dynamips
2022-08-11 23:36:29 INFO hypervisor.py:127 Starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_i5_log.txt', '-H', 'localhost:40385']
2022-08-11 23:36:29 INFO hypervisor.py:129 Dynamips process logging to /home/h4ck4u/Downloads/laRota/project-files/dynamips/dynamips_i5_stdout.txt
2022-08-11 23:36:29 INFO hypervisor.py:136 Dynamips process started PID=45765
2022-08-11 23:36:29 INFO __init__.py:305 Hypervisor localhost:40385 has successfully started
2022-08-11 23:36:29 INFO dynamips_hypervisor.py:92 Connected to Dynamips hypervisor on localhost:40385 after 0.0122 seconds
2022-08-11 23:36:29 INFO ethernet_switch.py:187 Ethernet switch "Switch2" [49a3f27d-0e22-4d88-a403-b057aec0090e] has been created
2022-08-11 23:36:29 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes HTTP/1.1" 201 1564 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO __init__.py:303 Creating new hypervisor localhost:52735 with working directory /home/h4ck4u/Downloads/laRota/project-files/dynamips
2022-08-11 23:36:29 INFO hypervisor.py:127 Starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_i6_log.txt', '-H', 'localhost:52735']
2022-08-11 23:36:29 INFO hypervisor.py:129 Dynamips process logging to /home/h4ck4u/Downloads/laRota/project-files/dynamips/dynamips_i6_stdout.txt
2022-08-11 23:36:29 INFO hypervisor.py:136 Dynamips process started PID=45781
2022-08-11 23:36:29 INFO __init__.py:305 Hypervisor localhost:52735 has successfully started
2022-08-11 23:36:29 INFO dynamips_hypervisor.py:92 Connected to Dynamips hypervisor on localhost:52735 after 0.0123 seconds
2022-08-11 23:36:29 INFO ethernet_switch.py:187 Ethernet switch "Switch3" [f74da918-d75d-47b5-ba96-294d52548818] has been created
2022-08-11 23:36:29 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes HTTP/1.1" 201 1564 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO __init__.py:303 Creating new hypervisor localhost:46375 with working directory /home/h4ck4u/Downloads/laRota/project-files/dynamips
2022-08-11 23:36:29 INFO hypervisor.py:127 Starting Dynamips: ['/usr/bin/dynamips', '-N1', '-l', 'dynamips_i7_log.txt', '-H', 'localhost:46375']
2022-08-11 23:36:29 INFO hypervisor.py:129 Dynamips process logging to /home/h4ck4u/Downloads/laRota/project-files/dynamips/dynamips_i7_stdout.txt
2022-08-11 23:36:29 INFO hypervisor.py:136 Dynamips process started PID=45797
2022-08-11 23:36:29 INFO __init__.py:305 Hypervisor localhost:46375 has successfully started
2022-08-11 23:36:29 INFO dynamips_hypervisor.py:92 Connected to Dynamips hypervisor on localhost:46375 after 0.0129 seconds
2022-08-11 23:36:29 INFO ethernet_switch.py:187 Ethernet switch "Switch4" [af532c9e-bfea-4634-b614-9c930b2d16f1] has been created
2022-08-11 23:36:29 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes HTTP/1.1" 201 1564 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO base_node.py:282 VPCS: PC1 [660e9f91-a906-4492-afe0-043e937ef232] created
2022-08-11 23:36:29 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/vpcs/nodes HTTP/1.1" 201 568 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO base_node.py:282 VPCS: PC2 [050e3150-51fc-46bd-9a0e-62730d86a3da] created
2022-08-11 23:36:29 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/vpcs/nodes HTTP/1.1" 201 568 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO vpcs_vm.py:376 VPCS "PC1" [660e9f91-a906-4492-afe0-043e937ef232]: NIO UDP added to port 0
2022-08-11 23:36:29 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/vpcs/nodes/660e9f91-a906-4492-afe0-043e937ef232/adapters/0/ports/0/nio HTTP/1.1" 201 381 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO ethernet_switch.py:275 Ethernet switch "Switch3" [f74da918-d75d-47b5-ba96-294d52548818]: NIO udp-c8109e09-b733-43dc-970c-2d53792364d0 bound to port 0
2022-08-11 23:36:29 INFO ethernet_switch.py:365 Ethernet switch "Switch3" [f74da918-d75d-47b5-ba96-294d52548818]: port 0 set as an access port in VLAN 1
2022-08-11 23:36:29 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/f74da918-d75d-47b5-ba96-294d52548818/adapters/0/ports/0/nio HTTP/1.1" 201 392 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO ethernet_switch.py:275 Ethernet switch "Switch3" [f74da918-d75d-47b5-ba96-294d52548818]: NIO udp-300687d2-e2be-4fac-98ef-db43b2849de3 bound to port 1
2022-08-11 23:36:29 INFO ethernet_switch.py:365 Ethernet switch "Switch3" [f74da918-d75d-47b5-ba96-294d52548818]: port 1 set as an access port in VLAN 1
2022-08-11 23:36:29 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/f74da918-d75d-47b5-ba96-294d52548818/adapters/0/ports/1/nio HTTP/1.1" 201 392 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:29 INFO nio_udp.py:74 NIO UDP udp-af000d89-87fd-4653-b9c1-616735b583b4 created with lport=10003, rhost=127.0.0.1, rport=10002
2022-08-11 23:36:29 INFO base_node.py:663 Starting new uBridge hypervisor localhost:37363
2022-08-11 23:36:29 INFO hypervisor.py:169 starting ubridge: ['/usr/bin/ubridge', '-H', 'localhost:37363']
2022-08-11 23:36:29 INFO hypervisor.py:171 logging to /home/h4ck4u/Downloads/laRota/project-files/dynamips/d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba/ubridge.log
2022-08-11 23:36:29 INFO hypervisor.py:179 ubridge started PID=45815
2022-08-11 23:36:29 INFO base_node.py:666 Hypervisor localhost:37363 has successfully started
2022-08-11 23:36:30 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:37363 after 0.1015 seconds
2022-08-11 23:36:30 INFO router.py:1291 Router "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba]: NIO udp-af000d89-87fd-4653-b9c1-616735b583b4 bound to port 0/1
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:29 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes/d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba/adapters/0/ports/1/nio HTTP/1.1" 201 385 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO nio_udp.py:74 NIO UDP udp-9db65b33-0bde-404f-9250-328ab7bd4f9f created with lport=10006, rhost=127.0.0.1, rport=10007
2022-08-11 23:36:30 INFO router.py:1291 Router "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba]: NIO udp-9db65b33-0bde-404f-9250-328ab7bd4f9f bound to port 0/0
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes/d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba/adapters/0/ports/0/nio HTTP/1.1" 201 385 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO ethernet_switch.py:275 Ethernet switch "Switch1" [8eb11374-ef90-4b72-aefe-567b232d1d27]: NIO udp-d1c0cd9a-2a61-4df6-8b60-3adaa83936a8 bound to port 0
2022-08-11 23:36:30 INFO ethernet_switch.py:365 Ethernet switch "Switch1" [8eb11374-ef90-4b72-aefe-567b232d1d27]: port 0 set as an access port in VLAN 1
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/8eb11374-ef90-4b72-aefe-567b232d1d27/adapters/0/ports/0/nio HTTP/1.1" 201 392 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO nio_udp.py:74 NIO UDP udp-9929c4ea-7820-4ec4-80c1-007d8f7e4d69 created with lport=10010, rhost=127.0.0.1, rport=10011
2022-08-11 23:36:30 INFO base_node.py:663 Starting new uBridge hypervisor localhost:44431
2022-08-11 23:36:30 INFO hypervisor.py:169 starting ubridge: ['/usr/bin/ubridge', '-H', 'localhost:44431']
2022-08-11 23:36:30 INFO hypervisor.py:171 logging to /home/h4ck4u/Downloads/laRota/project-files/dynamips/4e7aed76-6636-486a-85c8-0afc88731eb5/ubridge.log
2022-08-11 23:36:30 INFO hypervisor.py:179 ubridge started PID=45825
2022-08-11 23:36:30 INFO base_node.py:666 Hypervisor localhost:44431 has successfully started
2022-08-11 23:36:30 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:44431 after 0.1025 seconds
2022-08-11 23:36:30 INFO router.py:1291 Router "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5]: NIO udp-9929c4ea-7820-4ec4-80c1-007d8f7e4d69 bound to port 0/0
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes/4e7aed76-6636-486a-85c8-0afc88731eb5/adapters/0/ports/0/nio HTTP/1.1" 201 385 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO ethernet_switch.py:275 Ethernet switch "Switch1" [8eb11374-ef90-4b72-aefe-567b232d1d27]: NIO udp-5de1ef0f-a86f-4a04-bf99-2bf4b07cec7d bound to port 1
2022-08-11 23:36:30 INFO ethernet_switch.py:365 Ethernet switch "Switch1" [8eb11374-ef90-4b72-aefe-567b232d1d27]: port 1 set as an access port in VLAN 1
2022-08-11 23:36:30 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/8eb11374-ef90-4b72-aefe-567b232d1d27/adapters/0/ports/1/nio HTTP/1.1" 201 392 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO nio_udp.py:74 NIO UDP udp-9a48bce2-b63d-40ab-979b-13b672c03bcb created with lport=10014, rhost=127.0.0.1, rport=10015
2022-08-11 23:36:30 INFO router.py:1291 Router "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5]: NIO udp-9a48bce2-b63d-40ab-979b-13b672c03bcb bound to port 0/1
2022-08-11 23:36:30 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes/4e7aed76-6636-486a-85c8-0afc88731eb5/adapters/0/ports/1/nio HTTP/1.1" 201 385 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO ethernet_switch.py:275 Ethernet switch "Switch2" [49a3f27d-0e22-4d88-a403-b057aec0090e]: NIO udp-83dbaf73-5ec6-4318-8a79-5e677eed172b bound to port 0
2022-08-11 23:36:30 INFO ethernet_switch.py:365 Ethernet switch "Switch2" [49a3f27d-0e22-4d88-a403-b057aec0090e]: port 0 set as an access port in VLAN 1
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/49a3f27d-0e22-4d88-a403-b057aec0090e/adapters/0/ports/0/nio HTTP/1.1" 201 392 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO ethernet_switch.py:275 Ethernet switch "Switch2" [49a3f27d-0e22-4d88-a403-b057aec0090e]: NIO udp-c0940d9e-af7b-4e1e-887f-4f8a0b7623bb bound to port 1
2022-08-11 23:36:30 INFO ethernet_switch.py:365 Ethernet switch "Switch2" [49a3f27d-0e22-4d88-a403-b057aec0090e]: port 1 set as an access port in VLAN 1
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/49a3f27d-0e22-4d88-a403-b057aec0090e/adapters/0/ports/1/nio HTTP/1.1" 201 392 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO nio_udp.py:74 NIO UDP udp-20b5fedd-c3bd-4b24-8b54-addb2f4ec2a4 created with lport=10019, rhost=127.0.0.1, rport=10018
2022-08-11 23:36:30 INFO base_node.py:663 Starting new uBridge hypervisor localhost:40171
2022-08-11 23:36:30 INFO hypervisor.py:169 starting ubridge: ['/usr/bin/ubridge', '-H', 'localhost:40171']
2022-08-11 23:36:30 INFO hypervisor.py:171 logging to /home/h4ck4u/Downloads/laRota/project-files/dynamips/6bb93cba-99f2-4a54-b393-5ff244dd1e69/ubridge.log
2022-08-11 23:36:30 INFO hypervisor.py:179 ubridge started PID=45834
2022-08-11 23:36:30 INFO base_node.py:666 Hypervisor localhost:40171 has successfully started
2022-08-11 23:36:30 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:40171 after 0.1027 seconds
2022-08-11 23:36:30 INFO router.py:1291 Router "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69]: NIO udp-20b5fedd-c3bd-4b24-8b54-addb2f4ec2a4 bound to port 0/0
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes/6bb93cba-99f2-4a54-b393-5ff244dd1e69/adapters/0/ports/0/nio HTTP/1.1" 201 385 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO nio_udp.py:74 NIO UDP udp-97efd6be-141a-474f-8f58-68b121d434f3 created with lport=10022, rhost=127.0.0.1, rport=10023
2022-08-11 23:36:30 INFO router.py:1291 Router "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69]: NIO udp-97efd6be-141a-474f-8f58-68b121d434f3 bound to port 0/1
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes/6bb93cba-99f2-4a54-b393-5ff244dd1e69/adapters/0/ports/1/nio HTTP/1.1" 201 385 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO ethernet_switch.py:275 Ethernet switch "Switch4" [af532c9e-bfea-4634-b614-9c930b2d16f1]: NIO udp-50e6eb3e-0b12-4481-8623-e62ddde4a486 bound to port 0
2022-08-11 23:36:30 INFO ethernet_switch.py:365 Ethernet switch "Switch4" [af532c9e-bfea-4634-b614-9c930b2d16f1]: port 0 set as an access port in VLAN 1
2022-08-11 23:36:30 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/af532c9e-bfea-4634-b614-9c930b2d16f1/adapters/0/ports/0/nio HTTP/1.1" 201 392 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ports/udp HTTP/1.1" 201 246 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO ethernet_switch.py:275 Ethernet switch "Switch4" [af532c9e-bfea-4634-b614-9c930b2d16f1]: NIO udp-d6a5378f-e217-4507-8e42-4988c48ad9c4 bound to port 1
2022-08-11 23:36:30 INFO ethernet_switch.py:365 Ethernet switch "Switch4" [af532c9e-bfea-4634-b614-9c930b2d16f1]: port 1 set as an access port in VLAN 1
2022-08-11 23:36:30 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/af532c9e-bfea-4634-b614-9c930b2d16f1/adapters/0/ports/1/nio HTTP/1.1" 201 392 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO vpcs_vm.py:376 VPCS "PC2" [050e3150-51fc-46bd-9a0e-62730d86a3da]: NIO UDP added to port 0
2022-08-11 23:36:30 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:36:30 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/vpcs/nodes/050e3150-51fc-46bd-9a0e-62730d86a3da/adapters/0/ports/0/nio HTTP/1.1" 201 381 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:36:30 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:28 +0000] "POST /v2/projects/load HTTP/1.1" 201 726 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:36:30 INFO project_handler.py:272 New client has connected to the notification stream for project ID 'b963e8c3-589d-4a9c-b535-bbf1907b6ef0' (WebSocket method)
2022-08-11 23:36:30 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:30 +0000] "GET /v2/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/nodes HTTP/1.1" 200 32946 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:36:30 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:30 +0000] "GET /v2/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/links HTTP/1.1" 200 10832 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:36:30 INFO web_log.py:206 ::1 [11/Aug/2022:23:36:30 +0000] "GET /v2/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/drawings HTTP/1.1" 200 208 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:37:19 INFO router.py:110 Creating a new ghost IOS instance
2022-08-11 23:37:19 INFO router.py:489 Router "Ghost" [d7e70020-992d-4ffb-88dc-b539f2191eec]: has a new IOS image set: "/home/h4ck4u/GNS3/images/IOS/c3725-adventerprisek9_ivs-mz.124-15.T14.image"
2022-08-11 23:37:19 INFO router.py:850 Router "Ghost" [d7e70020-992d-4ffb-88dc-b539f2191eec]: ghost status set to 1
2022-08-11 23:37:19 INFO router.py:810 Router "Ghost" [d7e70020-992d-4ffb-88dc-b539f2191eec]: ghost file set to "/home/h4ck4u/Downloads/laRota/project-files/dynamips/c3725-adventerprisek9_ivs-mz.124-15.T14.image-128.ghost"
2022-08-11 23:37:19 INFO router.py:298 router "Ghost" [d7e70020-992d-4ffb-88dc-b539f2191eec] has been started
2022-08-11 23:37:19 INFO router.py:328 Router "Ghost" [d7e70020-992d-4ffb-88dc-b539f2191eec] has been stopped
2022-08-11 23:37:19 INFO router.py:1638 Router "Ghost" [d7e70020-992d-4ffb-88dc-b539f2191eec] has been deleted (including associated files)
2022-08-11 23:37:19 INFO router.py:850 Router "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba]: ghost status set to 2
2022-08-11 23:37:19 INFO router.py:810 Router "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba]: ghost file set to "/home/h4ck4u/Downloads/laRota/project-files/dynamips/c3725-adventerprisek9_ivs-mz.124-15.T14.image-128.ghost"
2022-08-11 23:37:19 INFO router.py:850 Router "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5]: ghost status set to 2
2022-08-11 23:37:19 INFO router.py:810 Router "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5]: ghost file set to "/home/h4ck4u/Downloads/laRota/project-files/dynamips/c3725-adventerprisek9_ivs-mz.124-15.T14.image-128.ghost"
2022-08-11 23:37:19 INFO router.py:850 Router "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69]: ghost status set to 2
2022-08-11 23:37:19 INFO router.py:810 Router "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69]: ghost file set to "/home/h4ck4u/Downloads/laRota/project-files/dynamips/c3725-adventerprisek9_ivs-mz.124-15.T14.image-128.ghost"
2022-08-11 23:37:20 INFO router.py:298 router "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5] has been started
2022-08-11 23:37:20 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:37:19 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes/4e7aed76-6636-486a-85c8-0afc88731eb5/start HTTP/1.1" 204 193 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:37:20 INFO router.py:298 router "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba] has been started
2022-08-11 23:37:20 INFO web_log.py:206 ::1 [11/Aug/2022:23:37:19 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes/d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba/start HTTP/1.1" 204 193 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:37:20 INFO router.py:298 router "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69] has been started
2022-08-11 23:37:20 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:37:19 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes/6bb93cba-99f2-4a54-b393-5ff244dd1e69/start HTTP/1.1" 204 193 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:37:20 INFO web_log.py:206 ::1 [11/Aug/2022:23:37:20 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/8eb11374-ef90-4b72-aefe-567b232d1d27/start HTTP/1.1" 204 200 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:37:20 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:37:20 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/f74da918-d75d-47b5-ba96-294d52548818/start HTTP/1.1" 204 200 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:37:20 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:37:20 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/49a3f27d-0e22-4d88-a403-b057aec0090e/start HTTP/1.1" 204 200 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:37:20 INFO web_log.py:206 ::1 [11/Aug/2022:23:37:20 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/af532c9e-bfea-4634-b614-9c930b2d16f1/start HTTP/1.1" 204 200 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:37:20 INFO base_node.py:830 VPCS: 'PC1' [660e9f91-a906-4492-afe0-043e937ef232]:local UDP tunnel created between port 10028 and 10029
2022-08-11 23:37:20 INFO vpcs_vm.py:225 Starting VPCS: ['/usr/bin/vpcs', '-p', '5008', '-m', '0', '-i', '1', '-F', '-R', '-s', '10028', '-c', '10029', '-t', '127.0.0.1', 'startup.vpc']
2022-08-11 23:37:20 INFO vpcs_vm.py:227 Logging to /home/h4ck4u/Downloads/laRota/project-files/vpcs/660e9f91-a906-4492-afe0-043e937ef232/vpcs.log
2022-08-11 23:37:20 INFO base_node.py:663 Starting new uBridge hypervisor localhost:35111
2022-08-11 23:37:20 INFO base_node.py:830 VPCS: 'PC2' [050e3150-51fc-46bd-9a0e-62730d86a3da]:local UDP tunnel created between port 10030 and 10031
2022-08-11 23:37:20 INFO vpcs_vm.py:225 Starting VPCS: ['/usr/bin/vpcs', '-p', '5010', '-m', '1', '-i', '1', '-F', '-R', '-s', '10030', '-c', '10031', '-t', '127.0.0.1', 'startup.vpc']
2022-08-11 23:37:20 INFO vpcs_vm.py:227 Logging to /home/h4ck4u/Downloads/laRota/project-files/vpcs/050e3150-51fc-46bd-9a0e-62730d86a3da/vpcs.log
2022-08-11 23:37:20 INFO base_node.py:663 Starting new uBridge hypervisor localhost:34419
2022-08-11 23:37:20 INFO hypervisor.py:169 starting ubridge: ['/usr/bin/ubridge', '-H', 'localhost:35111']
2022-08-11 23:37:20 INFO hypervisor.py:171 logging to /home/h4ck4u/Downloads/laRota/project-files/vpcs/660e9f91-a906-4492-afe0-043e937ef232/ubridge.log
2022-08-11 23:37:20 INFO hypervisor.py:179 ubridge started PID=45911
2022-08-11 23:37:20 INFO base_node.py:666 Hypervisor localhost:35111 has successfully started
2022-08-11 23:37:20 INFO hypervisor.py:169 starting ubridge: ['/usr/bin/ubridge', '-H', 'localhost:34419']
2022-08-11 23:37:20 INFO hypervisor.py:171 logging to /home/h4ck4u/Downloads/laRota/project-files/vpcs/050e3150-51fc-46bd-9a0e-62730d86a3da/ubridge.log
2022-08-11 23:37:20 INFO hypervisor.py:179 ubridge started PID=45913
2022-08-11 23:37:20 INFO base_node.py:666 Hypervisor localhost:34419 has successfully started
2022-08-11 23:37:20 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:35111 after 0.1030 seconds
2022-08-11 23:37:20 INFO ubridge_hypervisor.py:83 Connected to uBridge hypervisor on localhost:34419 after 0.1026 seconds
2022-08-11 23:37:20 INFO vpcs_vm.py:246 VPCS instance PC1 started PID=45895
2022-08-11 23:37:20 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:37:20 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/vpcs/nodes/660e9f91-a906-4492-afe0-043e937ef232/start HTTP/1.1" 200 659 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:37:20 INFO vpcs_vm.py:246 VPCS instance PC2 started PID=45903
2022-08-11 23:37:20 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:37:20 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/vpcs/nodes/050e3150-51fc-46bd-9a0e-62730d86a3da/start HTTP/1.1" 200 659 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:37:20 INFO web_log.py:206 ::1 [11/Aug/2022:23:37:19 +0000] "POST /v2/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/nodes/start HTTP/1.1" 204 166 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:37:20 INFO web_log.py:206 ::1 [11/Aug/2022:23:37:20 +0000] "GET /v2/computes HTTP/1.1" 200 1028 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:37:50 INFO router.py:1599 saving startup-config to /home/h4ck4u/Downloads/laRota/project-files/dynamips/4e7aed76-6636-486a-85c8-0afc88731eb5/configs/i2_startup-config.cfg
2022-08-11 23:37:50 INFO router.py:1599 saving startup-config to /home/h4ck4u/Downloads/laRota/project-files/dynamips/4e7aed76-6636-486a-85c8-0afc88731eb5/configs/i2_startup-config.cfg
2022-08-11 23:37:50 INFO router.py:1599 saving startup-config to /home/h4ck4u/Downloads/laRota/project-files/dynamips/d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba/configs/i1_startup-config.cfg
2022-08-11 23:37:50 INFO router.py:1599 saving startup-config to /home/h4ck4u/Downloads/laRota/project-files/dynamips/d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba/configs/i1_startup-config.cfg
2022-08-11 23:37:50 INFO router.py:1599 saving startup-config to /home/h4ck4u/Downloads/laRota/project-files/dynamips/6bb93cba-99f2-4a54-b393-5ff244dd1e69/configs/i3_startup-config.cfg
2022-08-11 23:37:50 INFO router.py:1599 saving startup-config to /home/h4ck4u/Downloads/laRota/project-files/dynamips/6bb93cba-99f2-4a54-b393-5ff244dd1e69/configs/i3_startup-config.cfg
2022-08-11 23:38:36 INFO notification_handler.py:82 Client has disconnected from controller WebSocket
2022-08-11 23:38:36 INFO server_handler.py:53 Start shutting down the server
2022-08-11 23:38:36 INFO router.py:328 Router "R3" [6bb93cba-99f2-4a54-b393-5ff244dd1e69] has been stopped
2022-08-11 23:38:36 INFO router.py:328 Router "R1" [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba] has been stopped
2022-08-11 23:38:36 INFO router.py:1599 saving startup-config to /home/h4ck4u/Downloads/laRota/project-files/dynamips/6bb93cba-99f2-4a54-b393-5ff244dd1e69/configs/i3_startup-config.cfg
2022-08-11 23:38:36 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:38:36 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes/6bb93cba-99f2-4a54-b393-5ff244dd1e69/stop HTTP/1.1" 204 192 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:38:36 INFO router.py:1599 saving startup-config to /home/h4ck4u/Downloads/laRota/project-files/dynamips/d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba/configs/i1_startup-config.cfg
2022-08-11 23:38:36 INFO web_log.py:206 ::1 [11/Aug/2022:23:38:36 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes/d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba/stop HTTP/1.1" 204 192 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:38:36 INFO router.py:328 Router "R2" [4e7aed76-6636-486a-85c8-0afc88731eb5] has been stopped
2022-08-11 23:38:36 INFO router.py:1599 saving startup-config to /home/h4ck4u/Downloads/laRota/project-files/dynamips/4e7aed76-6636-486a-85c8-0afc88731eb5/configs/i2_startup-config.cfg
2022-08-11 23:38:36 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:38:36 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/dynamips/nodes/4e7aed76-6636-486a-85c8-0afc88731eb5/stop HTTP/1.1" 204 192 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:38:36 INFO web_log.py:206 ::1 [11/Aug/2022:23:38:36 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/8eb11374-ef90-4b72-aefe-567b232d1d27/stop HTTP/1.1" 204 199 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:38:36 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:38:36 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/49a3f27d-0e22-4d88-a403-b057aec0090e/stop HTTP/1.1" 204 199 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:38:36 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:38:36 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/f74da918-d75d-47b5-ba96-294d52548818/stop HTTP/1.1" 204 199 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:38:36 INFO base_node.py:677 Stopping uBridge hypervisor localhost:35111
2022-08-11 23:38:36 INFO hypervisor.py:208 Stopping uBridge process PID=45911
2022-08-11 23:38:36 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:38:36 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/ethernet_switch/nodes/af532c9e-bfea-4634-b614-9c930b2d16f1/stop HTTP/1.1" 204 199 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:38:36 INFO base_node.py:677 Stopping uBridge hypervisor localhost:34419
2022-08-11 23:38:36 INFO hypervisor.py:208 Stopping uBridge process PID=45913
2022-08-11 23:38:36 INFO vpcs_vm.py:308 Stopping VPCS instance PC1 PID=45895
2022-08-11 23:38:36 INFO vpcs_vm.py:308 Stopping VPCS instance PC2 PID=45903
2022-08-11 23:38:36 INFO vpcs_vm.py:262 VPCS process has stopped, return code: 0
2022-08-11 23:38:36 INFO web_log.py:206 ::1 [11/Aug/2022:23:38:36 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/vpcs/nodes/660e9f91-a906-4492-afe0-043e937ef232/stop HTTP/1.1" 204 188 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:38:36 INFO vpcs_vm.py:262 VPCS process has stopped, return code: 0
2022-08-11 23:38:36 INFO web_log.py:206 127.0.0.1 [11/Aug/2022:23:38:36 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/vpcs/nodes/050e3150-51fc-46bd-9a0e-62730d86a3da/stop HTTP/1.1" 204 188 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:38:36 INFO base_node.py:331 VPCS: 'PC1' [660e9f91-a906-4492-afe0-043e937ef232]: is closing
2022-08-11 23:38:36 INFO base_node.py:331 VPCS: 'PC2' [050e3150-51fc-46bd-9a0e-62730d86a3da]: is closing
2022-08-11 23:38:36 INFO base_node.py:331 Dynamips: 'R1' [d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba]: is closing
2022-08-11 23:38:36 INFO base_node.py:331 Dynamips: 'R3' [6bb93cba-99f2-4a54-b393-5ff244dd1e69]: is closing
2022-08-11 23:38:36 INFO base_node.py:331 Dynamips: 'R2' [4e7aed76-6636-486a-85c8-0afc88731eb5]: is closing
2022-08-11 23:38:36 INFO base_node.py:677 Stopping uBridge hypervisor localhost:37363
2022-08-11 23:38:36 INFO hypervisor.py:208 Stopping uBridge process PID=45815
2022-08-11 23:38:36 INFO base_node.py:677 Stopping uBridge hypervisor localhost:40171
2022-08-11 23:38:36 INFO hypervisor.py:208 Stopping uBridge process PID=45834
2022-08-11 23:38:36 INFO base_node.py:677 Stopping uBridge hypervisor localhost:44431
2022-08-11 23:38:36 INFO hypervisor.py:208 Stopping uBridge process PID=45825
2022-08-11 23:38:36 INFO router.py:1599 saving startup-config to /home/h4ck4u/Downloads/laRota/project-files/dynamips/4e7aed76-6636-486a-85c8-0afc88731eb5/configs/i2_startup-config.cfg
2022-08-11 23:38:36 INFO hypervisor.py:148 Stopping Dynamips process PID=45717
2022-08-11 23:38:36 INFO ethernet_switch.py:252 Ethernet switch "Switch2" [49a3f27d-0e22-4d88-a403-b057aec0090e] has been deleted
2022-08-11 23:38:36 INFO hypervisor.py:148 Stopping Dynamips process PID=45765
2022-08-11 23:38:36 INFO ethernet_switch.py:252 Ethernet switch "Switch3" [f74da918-d75d-47b5-ba96-294d52548818] has been deleted
2022-08-11 23:38:36 INFO hypervisor.py:148 Stopping Dynamips process PID=45781
2022-08-11 23:38:36 INFO ethernet_switch.py:252 Ethernet switch "Switch1" [8eb11374-ef90-4b72-aefe-567b232d1d27] has been deleted
2022-08-11 23:38:36 INFO hypervisor.py:148 Stopping Dynamips process PID=45749
2022-08-11 23:38:36 INFO ethernet_switch.py:252 Ethernet switch "Switch4" [af532c9e-bfea-4634-b614-9c930b2d16f1] has been deleted
2022-08-11 23:38:36 INFO hypervisor.py:148 Stopping Dynamips process PID=45797
2022-08-11 23:38:36 INFO router.py:1599 saving startup-config to /home/h4ck4u/Downloads/laRota/project-files/dynamips/d013b6d9-c1e4-4d80-94f0-b7ed8e1a01ba/configs/i1_startup-config.cfg
2022-08-11 23:38:36 INFO hypervisor.py:148 Stopping Dynamips process PID=45701
2022-08-11 23:38:36 INFO router.py:1599 saving startup-config to /home/h4ck4u/Downloads/laRota/project-files/dynamips/6bb93cba-99f2-4a54-b393-5ff244dd1e69/configs/i3_startup-config.cfg
2022-08-11 23:38:36 INFO hypervisor.py:148 Stopping Dynamips process PID=45733
2022-08-11 23:38:37 INFO project.py:361 Project b963e8c3-589d-4a9c-b535-bbf1907b6ef0 with path '/home/h4ck4u/Downloads/laRota' closed
2022-08-11 23:38:37 INFO web_log.py:206 ::1 [11/Aug/2022:23:38:36 +0000] "POST /v2/compute/projects/b963e8c3-589d-4a9c-b535-bbf1907b6ef0/close HTTP/1.1" 204 168 "-" "Python/3.10 aiohttp/3.8.1"
2022-08-11 23:38:37 INFO web_log.py:206 ::1 [11/Aug/2022:23:38:36 +0000] "POST /v2/shutdown HTTP/1.1" 201 197 "-" "GNS3 QT Client v2.2.33.1"
2022-08-11 23:38:37 INFO web_server.py:113 Closing 2 websocket connections...
2022-08-11 23:38:37 INFO compute.py:460 Connection closed to compute 'local' WebSocket 'http://localhost:3080/v2/compute/notifications/ws'
2022-08-11 23:38:37 INFO project_handler.py:281 Client has disconnected from notification stream for project ID 'b963e8c3-589d-4a9c-b535-bbf1907b6ef0' (WebSocket method)
2022-08-11 23:38:37 INFO notification_handler.py:59 Client has disconnected from compute WebSocket
2022-08-11 23:38:37 INFO __init__.py:138 Controller is stopping
2022-08-11 23:38:37 INFO run.py:272 Remove PID file /home/h4ck4u/.config/GNS3/2.2/gns3_server.pid
